apps/backend/src/mcp/tools/container_management.py

```
@@ -895,130 +879,153 @@ async def get_service_dependencies(
+                                    "shared_network": network_name,
+                                    "relationship": "network_peer",
+                                }
+                            )
+
             # Check volume-based dependencies
-            target_volumes = set(mount.get("Source", "") for mount in target_mounts if mount.get("Type") in ["bind", "volume"])
+            target_volumes = set(
+                mount.get("Source", "")
+                for mount in target_mounts
+                if mount.get("Type") in ["bind", "volume"]
+            )
```

Ruff (0.12.2)

Unnecessary generator (rewrite as a set comprehension)

Rewrite as a set comprehension

(C401)